kafka/changestream/twitter_user_names-filter/main.go

```diff
@@ -0,0 +1,43 @@
+package main
+
+import (
+	"encoding/json"
+
+	"github.com/codeuniversity/smag-mvp/kafka/changestream"
+	"github.com/codeuniversity/smag-mvp/service"
+	"github.com/codeuniversity/smag-mvp/utils"
+	"github.com/segmentio/kafka-go"
+)
+
+func main() {
+	kafkaAddress := utils.GetStringFromEnvWithDefault("KAFKA_ADDRESS", "127.0.0.1:9092")
+	groupID := utils.MustGetStringFromEnv("KAFKA_GROUPID")
+	changesTopic := utils.GetStringFromEnvWithDefault("KAFKA_CHANGE_TOPIC", "postgres.public.twitter_users")
+	namesTopic := utils.GetStringFromEnvWithDefault("KAFKA_NAME_TOPIC", "twitter-user_names")
+
+	f := changestream.NewFilter(kafkaAddress, groupID, changesTopic, namesTopic, filterChange)
+
+	service.CloseOnSignal(f)
+	waitUntilClose := f.Start()
+
+	waitUntilClose()
+}
+
+type user struct {
```

Comment:
If it is not possible of different column names in the table, still move it to models and we unify that in a separate issue.

Reply:
The user structs within the filters differ by their struct tags because they have different column names in the database. Therefore, we would need to alter the `user_name` database column to change this. However, I would also not move it to `models` because it would create some naming issues and is very clean in the current version with private structs.